ci/travis/build-docker.sh

```diff
@@ -83,11 +83,15 @@ if [ "${USE_LIBCXX:-}" = "yes" ]; then
   cmake_extra_flags+=( "-DGOOGLE_CLOUD_CPP_USE_LIBCXX=ON" )
 fi
 
-# We want ${CMAKE_FLAGS} to expand as separate arguments.
+# We add ${cmake_extra_flags[@]} to itself is because set -u doesn't like empty
```

Comment:
I'm not sure I understand "add to itself" here.  Is that a remnant of a different workaround?

Reply:
No, I just worded it poorly. Fixed.